src/main/scala/org/allenai/plugins/NodeJsPlugin.scala

```diff
@@ -113,26 +92,56 @@ object NodeJsPlugin extends AutoPlugin {
     // we don't care if the command fails here because we are just
     // passing through to the local `npm`
     try {
-      val (newState, env) = extracted.runTask(environment in Npm, state)
+      val env = extracted.getOpt(environment in Npm).get
       exec(args.mkString(" "), extracted.getOpt(nodeProjectDir in Npm).get, env)
-      newState
+      state
     } catch {
       case NpmMissingException => state.fail
       case t: Throwable => state
     }
   }
 
+  /** @returns the node build environment variables, with the given NODE_ENV and NODE_BUILD_DIR set
+    */
+  def getEnvironment(nodeEnv: String, buildDir: File): Map[String, String] = {
+    Map(
+      "NODE_ENV" -> nodeEnv,
+      "NODE_API_HOST" -> "/api",
```

Comment:
Seems like this should be configurable too, just in case someone decides to use a different API root.   Though I also hope to eliminate this property in the front-end build system, so maybe we leave it as is for now.